src/systems/filecoin_vm/runtime/runtime.go

```diff
@@ -571,6 +590,16 @@ func (rt *VMContext) Compute(f ComputeFunctionID, args []Any) Any {
 	return def.Body(args)
 }
 
+func (rt *VMContext) GetActorCodeID(actorAddr addr.Address) (ret actor.CodeID, ok bool) {
+	IMPL_FINISH()
+	panic("")
+}
+
+func (rt *VMContext) GetMinerOwnerAddr(actorAddr addr.Address) (ownerAddr addr.Address, ok bool) {
```

Comment:
This would require an import cycle in order for the StoragePowerActor to call StorageMinerActor methods and vice versa, which isn't possible without some significant refactoring. For the time being, I'll replace `GetMinerOwnerAddr` with a TODO.